README.md

```diff
@@ -98,3 +98,83 @@ resource "aws_ami_from_instance" "example" {
 
 ## References
 * Thanks to https://github.com/cloudposse/tf_bastion for the inspiration
```

Comment:
fix repo name `tf_bastion`

Reply:
It's also kind of strange to say "thanks" to ourselves. Let's just link to the repo, but drop the praise =)